recipes/limix-legacy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "limix-legacy" %}
+{% set version = "0.8.10" %}
+{% set sha256 = "9e9e5a66283272536001d8f6fc90c4b36ae9d9b0cc34a63017549f76050eecac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Does this package explicitly not support windows or has it just never been tested? If it's the latter, perhaps allow appveyor to attempt to build the package to see what happens.